spec/js/ko-components/view-models/orifice-calculator-spec.coffee

```diff
@@ -8,19 +35,41 @@ describe 'orifice-calculator-viewmodel-spec', ->
     it 'should have default value of "1.939\'\' XS, Sch 80, Sch 80S"', ->
       expect(viewModel.selectedPipeID()).toEqual '1.939\'\' XS, Sch 80, Sch 80S'
 
+  describe 'operatingPressure', ->
+    beforeEach ->
+      field = viewModel.operatingPressure
+
+    itBehavesLikeMandatoryField()
+    itBehavesLikeIntegerField()
+
   describe 'operatingPressureRead', ->
     it 'should have "Gauge" and "Absolute"', ->
         expect(viewModel.operatingPressureRead()).toEqual ['Gauge','Absolute']
 
     it 'should have default value of "Gauge"', ->
       expect(viewModel.chosenOperatingPressureRead()).toEqual 'Gauge'
 
-  describe 'operatingPressureUnits', ->
-    it 'should have intialized the input data', ->
-      expect(viewModel.operatingPressureUnits().length).toEqual 8
+  describe 'baseSpecificGravity', ->
+    beforeEach ->
+      field = viewModel.baseSpecificGravity
+      validationMessage = 'Please enter the base specific gravity'
```

Comment:
True. Testing the validation message is testing the functionality of Knockout Validation, which is redundant.

Done.